.github/workflows/smoke-tests.yml

```diff
@@ -0,0 +1,53 @@
+name: Smoke Tests
```

Comment:
should this be an action rather than a workflow? Seems like it is being dispatched from the canary and manual workflow